src/Integration.Vsix.UnitTests/SonarLintTagger/TextBufferIssueTrackerTests.cs

```diff
@@ -129,27 +131,19 @@ public void Ctor_CreatesFactoryWithCorrectData()
         }
 
         [TestMethod]
-        public void Ctor_FailsToRetrieveProjectGuid_NonCriticalException_ExceptionCaught()
```

Comment:
Previously the analysis was requested synchronously so we needed to catch exceptions for the initial analysis requested in the constructor to prevent crashes.

Now, the constructor still makes the analysis request, but it is handle asynchronously and exceptions aren't raised. I added a new test to check this.